Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -41,13 +41,11 @@ private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPla
 
 		public static List<int> FindAllIndex(int[] elements, List<int> indexLists, int elem) {
 			/* Generate all index of an element */
-			List<int> indexList = new List<int>();
-
-			foreach (int i in indexLists) {
-				if (elements[i] == elem) 
-				indexList.Add(i);
-			}
 
+			var query = from i in indexLists 

```

Comment:
this is correct but can be done in single statement using where function of linq